migrations/20180213214150_create_sessions_table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = (knex, Promise) =>
```

Comment:
what does 'knex' refer to? generated vs a name you selected?

Reply:
knex is the database query builder library that support multiple relational databases. [Knex Project](http://knexjs.org/).  All the node ORM packages use knex.